### ISSUE TITLE
sitewide - 508-defect-2 – Progress Button - Make Find VA Benefits work with JAWS+IE11

### ISSUE THREAD

**Issue by erikphansen** [department-of-veterans-affairs/va.gov-team#7324]
There is an issue with JAWS re-reading the Find VA Benefits page when you update the selected options and navigate back to the main My VA Dashboard. We should work with Jennifer Strickland and/or Trevor Pierce to iterate on possible fixes.

**Comment by erikphansen** [comment]
Step one was to stop using a `LoadingButton` and also stop dynamically enabling/disabling the submit button based on if the currently selected benefits match what's currently saved for that user. That seems to have alleviated the issue with IE11+JAWS. But it's not a real solution because we want to have the `LoadingButton` in place for the majority of users who are not using IE11+JAWS.

Next steps would be to refactor the `LoadingButton` itself to hopefully make it work better with IE11+JAWS

- When the `LoadingButton` switches to the spinner state we set it to be `aria-hidden="true"` and `role="presentation"`
- and also add a "loading" message with our `.sr-only` tag so it's visible to screen readers but not visible in the browser?

**Comment by jenstrickland** [comment]
> Note: this issue also happens when a user logs in via ID.me, where there is a button similar to the progress button. After logging in, the screen reader continues to read the previous login pages, although the page shown in the browser is the landing page.

**Comment by Samara-Strauss** [comment]
Closing this since we removed this feature for the new dashboard.
